build/com.liferay.ide-repository/bundle/build.xml

```diff
@@ -285,6 +285,36 @@
         </exec>
     </target>
 
+    <target name="update-feature">
+        <if>
+            <not>
+                <equals arg1="${update.features}" arg2="" />
+            </not>
+            <then>
+                <exec executable="${builder.exe}" outputproperty="output" failonerror="true" dir="${target.dir}">
+                    <arg value="-nosplash" />
+                    <arg value="-application" />
+                    <arg value="org.eclipse.equinox.p2.director" />
+                    <arg value="-uninstallIU" />
```

Comment:
You are uninstalling instead of updating.  SO this target shouldn't be called update.